monai/handlers/mlflow_handler.py

```diff
@@ -86,6 +93,11 @@ def __init__(
         global_epoch_transform: Callable = lambda x: x,
         state_attributes: Optional[Sequence[str]] = None,
         tag_name: str = DEFAULT_TAG,
+        experiment_name: str = "default_experiment",
+        run_name: str = "test_run",
```

Comment:
How about automatically generating the `run_name` by datetime/model name, if user did not specify it in the `__init__`? For example, the user doesn't need to provide inputs but they can finally see their experiments in different `run_name`, instead of "test_run" in the console.